docs/developer/validation-rules/resource-model-validation.md

```diff
@@ -0,0 +1,38 @@
+# Identifying resource model definitions in a specification
+## Description
+A model definition is considered as a resource model definition if:
+
+- For each model, check if that model is returned by a PUT operation as a part of 200 or 201 response codes.
+- If the model has an 'allOf' Resource at any level [name of the root model is 'Resource'] AND the model is returned by a GET operation in 200 response.
+- If the model has 'x-ms-azure-resource': true or if any model has an 'x-ms-azure-resource' set to true in its model hierarchy and is not named either of 'Resource', 'TrackedResource' or 'ProxyResource', consider this as a resource
```

Comment:
remove the last part of the sentence? "consider this as a resource" , seems redundant.